CHANGES/5847.bugfix

```diff
@@ -0,0 +1 @@
+Extended repo metadata with relative_path to avoid sync issue with repositories contains productid.
```

Comment:
This is fine as a commit message, however I suggest to make the changelog entry more user-friendly.
Adding new field to a model is an implementation detail (it's not exposed in the API), so users are likely not interested in that.
How about "Fixed the sync issue for repositories with the same metadata files but different filenames. E.g. productid in RHEL8 BaseOS and Appstream."